src/services/domainService.js

```diff
@@ -12,7 +12,7 @@ const { getCustomConfigs } = require('./application/custom');
 let myIP = null;
 let myFDMnameORip = null;
 
-const mandatoryApps = ['explorer', 'KDLaunch', 'website', 'Kadena3', 'Kadena4', 'HavenNodeMainnet'];
+const mandatoryApps = [];
```

Comment:
Why are we removing the mandatory apps here? Are these not mandatory anymore?

Reply:
@TheTrunk actually do you know why we have those mandatory apps?

@fegauthier, good idea, Ill add mandatoryApps to config file once we verify that we still need them

Reply:
Yes mandatory apps are needed as we can get a bad node from fluxos feeding this fdm. Or some app expired on flux that should not expire but some instnace still may be running - in this case we are getting some notifications that fdm failed to reload and we can quickly fix that.